templates/initial-setup/admin-creation.php

```diff
@@ -22,19 +22,19 @@
       <?php } ?>
 
 
-      <form action="options.php" method="POST" onsubmit="onNext()">
+      <form action="options.php" method="POST">
 
         <div class="row">
           <div class="a0-admin-creation col-sm-6 col-xs-10">
-            <input type="text" id="admin-email" value="<?php echo $current_user->user_email; ?>" disabled />
-            <input type="password" id="admin-password" name="admin-password" placeholder="Password" value="" />
+            <input type="text" id="admin-email" value="<?php echo $current_user->user_email; ?>" disabled>
+            <input type="password" id="admin-password" name="admin-password" placeholder="Password" value="" required>
```

Comment:
Don't let the form submit if we don't have a password

Reply:
Just the password is not enough, but that's a beginning. What about email/username?

Reply:
It uses the current admin email address, next line above.